run.bat

```diff
@@ -0,0 +1 @@
+hugo.exe serve --source "%cd%\src"
```

Comment:
There's already vscode tasks for this, and also, this is only for Windows, what about mac and Linux ;)

Reply:
will remove